tools/debug-ui/src/LogFileDrop.tsx

```diff
@@ -65,7 +86,15 @@ export const LogFileDrop = ({ onFileDrop }: LogFileDropProps) => {
                 Drag and drop a file here
             </div>
             <input type="file" onChange={handleClick} />
-            <button>Upload</button>
+            {segments !== null && segments.length > 0 && <div>
+                <h3>There are multiple TestLoop runs within the same test. Choose one:</h3>
+                {segments.map((segment, i) => (
```

Comment:
I should probably wait for the demo, but anyway - is it expected to support multiple tests or multiple runs of the same test?
Because in case of the latter, how can I map the test to the log number (I assume that's what `i` is for)?

Reply:
Yes that's what the i is for. The UI will display multile buttons that each says "Log #x"